src/Cameras/VR/babylon.vrExperienceHelper.ts

```diff
@@ -116,7 +116,17 @@ module BABYLON {
             this._laserPointer.rotation.x = Math.PI / 2;
             this._laserPointer.position.z = -0.5;
             this._laserPointer.isVisible = false;
-            this._laserPointer.parent = webVRController.mesh;
+
+            if(!webVRController.mesh){
+                // Create an empty mesh that is used prior to loading the high quality model
+                var preloadMesh = new Mesh("preloadControllerMesh", scene);
+                var preloadPointerPose = new Mesh("POINTING_POSE", scene);

```

Comment:
why is the naming different? Is it being used later, or just to emphasis usage?

Reply:
This was just to emphasis usage.